okio/src/commonMain/kotlin/okio/ByteString.kt

```diff
@@ -133,6 +133,14 @@ internal constructor(data: ByteArray) : Comparable<ByteString> {
    */
   fun rangeEquals(offset: Int, other: ByteArray, otherOffset: Int, byteCount: Int): Boolean
 
+  /**
+   * Copies bytes of this in `[offset..offset+byteCount]` to other in
+   * `[otherOffset..otherOffset+byteCount]`.
```

Comment:
Should this be `targetOffset` rather than `otherOffset`?

Reply:
YEP! Good catch

Reply:
Fixed!